include/caffe/layers/cudnn_conv_layer.hpp

```diff
@@ -34,7 +34,7 @@ class CuDNNConvolutionLayer : public ConvolutionLayer<Dtype> {
  public:
   explicit CuDNNConvolutionLayer(const LayerParameter& param)
       : ConvolutionLayer<Dtype>(param), handles_setup_(false),
-        backward_passed_ctr_(0) {}
+        use_algo_seeker_(true), use_modest_workspace_(true) {}
```

Comment:
Is this too late to add `use_no_workspace` mode as per our discussion? If so, no problem, we'll add it in next release.